bnd.bnd

```diff
@@ -1,3 +1,7 @@
 ScriptEngine-Name:${project.name}
 ScriptEngine-Version:${project.version}
+Import-Package:     org.apache.sling.commons.compiler.*;resolution:=optional, \\
```

Comment:
I don't think that compiler and dynamic classloader can be optional, otherwise the JSP script factory would not load if the classes are missing

Reply:
Yes, you're right. Only the `org.apache.sling.scripting.bundle.tracker.*` package should be optional.